### ISSUE TITLE
How to sink playlists between YTM and spotify

### ISSUE THREAD

**Issue by Kshitij841** [platers/unitunes#26]
I have a spotify playlist with 140 ish songs and ytm with 1800+
How do I sync them. When I press Sync All it just pushes and pulls but my spotify playlist still has the same number of songs. I am probably missing something, could someone help me out?

![uni](https://user-images.githubusercontent.com/75718745/206836075-99788f85-c52c-4d08-a86d-54d4edf998cf.png)

**Comment by djlivooo** [comment]
Did you found a way to do that?
Because I'm in the same issue. I don't know how can I sync between them.

**Comment by Kshitij841** [comment]
> Did you found a way to do that? Because I'm in the same issue. I don't know how can I sync between them.

Not really, I gave up and used the online tool, using tune my music, did it 3-4 times on my playlist and made different playlist and then merged them by a script/tool or something from youtube, can not remember it perfectly.

I found this recently, https://github.com/yt2sp/yt2sp , I don't know if it will work or not but maybe you can try your luck here or just brute force it like me using tunemymusic.

**Comment by djlivooo** [comment]
> > Did you found a way to do that? Because I'm in the same issue. I don't know how can I sync between them.
> 
> Not really, I gave up and used the online tool, using tune my music, did it 3-4 times on my playlist and made different playlist and then merged them by a script/tool or something from youtube, can not remember it perfectly.
> 
> I found this recently, https://github.com/yt2sp/yt2sp , I don't know if it will work or not but maybe you can try your luck here or just brute force it like me using tunemymusic.

THANNKKKKKKKS. And a tool for the opposite? From Spotify to YT Music?

**Comment by Kshitij841** [comment]
The same could be done for Spt to YTM, the brute force. I never needed this, so you can google spotify to yt music GitHub/Reddit, multiple results should show up.
